cluster-autoscaler/core/scaledown/actuation/actuator_test.go

```diff
@@ -1191,8 +1349,18 @@ func removablePod(name string, node string) *apiv1.Pod {
 	}
 }
 
+func generateDsPods(count int, node string) []*apiv1.Pod {
+
+	var result []*apiv1.Pod
+	for i := 0; i < count; i++ {
+		name := fmt.Sprintf("ds-pod-%d", i)
+		result = append(result, generateDsPod(name, node))
+	}
+	return result
+}
+
 func generateDsPod(name string, node string) *apiv1.Pod {
-	pod := removablePod(name, node)
+	pod := removablePod(fmt.Sprintf("%s-%s", node, name), node)
```

Comment:
Earlier, we were using this function like this:
```
generateDsPod("drain-node-0-ds-pod-0", "drain-node-0")
```
And it would give us a DS pod with the name `drain-node-0-ds-pod-0`
Now, we use `generateDsPods` which calls `generateDsPod` like this:
```
generateDsPod("ds-pod-0", "drain-node-0")
```
This would give us the same result i.e., `drain-node-0-ds-pod-0`